README.md

```diff
@@ -98,16 +98,21 @@ Allows you to release updates to your iOS TestFlight beta app or production app
 
 5. **Email and Password** *(String, required if authentication method is `Username and Password`)* - Specify your Apple ID Developer account email and password here.
 
-6. **Bundle ID** *(String)* - Unique app identifier (e.g. com.myapp.etc).  The **Bundle ID** is only required if "Track" is *Production*.
-
-7. **Binary Path** *(File path, Required)* - Path to the IPA file you want to publish to the specified track.  A glob pattern can be used but it must resolve to exactly one IPA file.
-
 #### Release Options
 
 **Track** *(String, Required)* - Release track to publish the binary to (e.g. `TestFlight`  or `Production` ).
+##### Common Release Options (Available for TestFlight and Production track)
+
```

Comment:
Please update https://github.com/microsoft/app-store-vsts-extension/blob/master/docs/vsts-README.md also

Reply:
Thank you! Done